From 164ba2880ef2f126b69c24b1454a90122dc296d9 Mon Sep 17 00:00:00 2001
From: John Linn <john.linn@xilinx.com>
Date: Wed, 1 Aug 2012 17:53:22 -0700
Subject: [PATCH 594/628] Xilinx: ARM: remove initrd from device tree boot
 args

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 218a24f96ea78cc1698fd4ac2d3f505e079adf02

Now that bootm is being used in u-boot these are i
no longer needed and cause problems. The initrd is
needed if you are not booting the kernel from u-boot
(like from XMD) so you may need to add it back
in.

Signed-off-by: John Linn <john.linn@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 arch/arm/boot/dts/zynq-zc706.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/zynq-zc706.dts b/arch/arm/boot/dts/zynq-zc706.dts
index d4b36de..fb6520f 100644
--- a/arch/arm/boot/dts/zynq-zc706.dts
+++ b/arch/arm/boot/dts/zynq-zc706.dts
@@ -11,7 +11,7 @@
 		reg = <0x00000000 0x10000000>;
 	};
 	chosen {
-		bootargs = "console=ttyPS0,115200 root=/dev/ram rw initrd=0x800000,8M ip=:::::eth0:dhcp earlyprintk";
+		bootargs = "console=ttyPS0,115200 root=/dev/ram rw ip=:::::eth0:dhcp earlyprintk";
 		linux,stdout-path = "/amba@0/uart@E0001000";
 	};
 
-- 
1.7.5.4

